action.yml

```diff
@@ -170,6 +170,11 @@ runs:
           kubectl rollout status --watch --timeout 300s deployment/metrics-server -n kube-system
         fi
         if [[ "${{ inputs.traefik-enabled }}" == true ]]; then
+          # NOTE: Different versions of k3s install traefik in different ways,
+          #       by waiting for these jobs if they exist, we will be fine no
+          #       matter what.
```

Comment:
Do you know which versions use which method? If so it'd be helpful to include a comment here in case someone needs to debug other problems, and also to ease cleanup when we next drop an old K3s version.

Reply:
@manics I don't, I think its different patch versions of k3s releases rather than specific k8s versions etc.

I saw k8s 1.17 and 1.19 fail but 1.18 succeed if i recall correctly etc, and some installed Traefik v1, other v2, some installed via a HelmChart resource, others didn't.

Reply:
That sounds a bit messed up 😐

Reply:
There was no clear information when they changed HOW they install traefik, only that they install v2 in some later versions, but they started adopting a different way to install traefik in earlier versions as well so I don't see myself able to answer this so well.